packages/amplify-nodejs-function-template-provider/resources/lambda/serverless/package.json.ejs

```diff
@@ -5,8 +5,7 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "aws-serverless-express": "^3.3.5",
-    "body-parser": "^1.17.1",
+    "@vendia/serverless-express": "^4.10.4",
```

Comment:
why are we using 3rd party dependency for this?

Reply:
`aws-serverless-express` is being deprecated and rebranded to [vendia](https://www.npmjs.com/package/aws-serverless-express)
on the npm: On 11/30, the AWS Serverless Express library is moving to [Vendia](https://github.com/vendia) and will be rebranded to serverless-express. Similarly, the [aws-serverless-express NPM package](https://www.npmjs.com/package/aws-serverless-express) will be deprecated in favor of a new serverless-express package.